CONTRIBUTING.md

```diff
@@ -37,7 +37,14 @@ This is a rough outline of what a contributor's workflow looks like:
 - Make sure your commit messages are in the proper format (see below).
 - Push your changes to a topic branch in your fork of the repository.
 - Make sure the tests pass, and add any new tests as appropriate.
-- Submit a pull request to the original repository.
+- Submit a pull request to the original repository and assign it to yourself.
+- Submit a comment with the sole content "@reviewer PTAL" (please take another look) in GitHub
+  and replace "@reviewer" with the correct recipient.
+  Do not reassign the pull request.
+- When addressing pull request review comments add new commits to the existing pull request.
```

Comment:
Or if the new patches after a review are refactoring...

A good metric could be: if the new commits added after a review are about the same size as (or bigger than) the previous commits, then squashing is better.